tests/user_restricted/test_user_restricted.py

```diff
@@ -22,33 +22,8 @@ def test_retrieve_patient(self, headers_with_token):
             self.headers
         )
         helpers.check_response_headers(response, self.headers)
-        response_body = json.loads(response.text)
-        assert response.status_code == 200
-        # check id matches
-        assert response_body["id"] == retrieve[0]["patient"]
-        assert response_body["resourceType"] == "Patient"
-        # check the shape of response
-        assert response_body["address"] is not None
-        assert isinstance(response_body["address"], list)
-
-        assert response_body["birthDate"] is not None
-        assert isinstance(response_body["birthDate"], str)
-        assert len(response_body["birthDate"]) > 1
-
-        assert response_body["gender"] is not None
-        assert isinstance(response_body["gender"], str)
-
-        assert response_body["name"] is not None
-        assert isinstance(response_body["name"], list)
-        assert len(response_body["name"]) > 0
-
-        assert len(response_body["identifier"]) > 0
-        assert isinstance(response_body["identifier"], list)
-
-        assert len(response_body["extension"]) > 0
-        assert isinstance(response_body["extension"], list)
-
-        assert response_body["meta"] is not None
+        helpers.check_response_status_code(response, 200)
+        helpers.check_retrieve_response_body_shape(response)
```

Comment:
I think this has crept back in/is a reversion to previous behaviour - we probably want to keep it like it is on the left

Reply:
that was me, i moved the response body shape check into a seperate helper func for DRYness